Router.svelte

```diff
@@ -106,19 +106,36 @@ export const querystring = derived(
     ($loc) => $loc.querystring
 )
 
+
+/**
+ * Writable store that returns if the browser back button was clicked
```

Comment:
The word "store" in a Svelte context has a precise meaning, can you change it to something else?

Also, since this is not an exported property, please use a simple one-line comment `//` rather than a doc block (`/**`).